docs/job-item-flow.md

```diff
@@ -0,0 +1,15 @@
+# Information flow of a Job Item
+
+To make the best use of Redis, we propose you to configure your application to work in the manner below.
+
+Once a Job is submitted, the workload is prepared in Redis, who should have three queues: *main*, *processing* e *results*. These queues represent the state of each item put in them.
+
+Each item is put in the *main* queue in sequence, and this means they are ready to be fetched for processing. When a replica of the user's application fetches an item, it moves the item to the end of the *processing* queue, and stays there until the replica finishes processing it. After finishing, the item is deleted as the results produced are pushed in the *results* queue.
```

Comment:
"and stays there until" -> "which stays there until";

Reply:
Fixed in commit 59cdc4d104b8a0e7ba3f89aab68eac8ee3d714b7